spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/endpoint/NotificationMessageHandlerMethodArgumentResolver.java

```diff
@@ -36,6 +37,7 @@
 import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
 import org.springframework.util.StringUtils;
 import org.springframework.web.util.WebUtils;
+import sun.text.normalizer.UTF16;
```

Comment:
It's not used and for the future we cannot use classes from `sun.*` package.

Reply:
Removed. Idk why formatter hasn't removed it. 
import org.springframework.web.util.WebUtils; removed as well since it was not used.